lib_collections/models.py

```diff
@@ -217,6 +217,266 @@ class RelatedCollectionPagePlacement(Orderable, models.Model):
+    """
+    Intermediate class for metadata fields within a Collection Page
+    result.  
+    (needed to create an InlinePanel)
+
+    """
+    page = ParentalKey(
+        'lib_collections.CollectionPage', related_name="col_obj_metadata"
+    )
+
+
+class CResult(models.Model):
+    """
+    Class for search results within a Collection Page.
+    """
+    field_label = models.CharField(max_length=255, blank=True)
+    field_identifier = models.CharField(
+        max_length=255, blank=True, help_text="EDM/IIIF field identifier"
+    )
+
+    panels = [
+        FieldPanel('field_label'),
+        FieldPanel('field_identifier'),
+    ]
+
+    class Meta:
+        abstract = True
+
+
+class CollectionPageResult(Orderable, CResult):
+    """
+    Intermediate class for results within a Collection Page.  
+    (needed to create an InlinePanel)
+
+    """
+    page = ParentalKey(
+        'lib_collections.CollectionPage', related_name="col_result"
+    )
+
+
+class CFacet(models.Model):
+    """
+    Class for facets within a Collection Page.
+    """
+    label = models.CharField(max_length=255, blank=True)
+    search_handler_location = models.CharField(max_length=255, blank=True)
+    includes_ocr = models.BooleanField(
+        default=False, help_text='Does this include OCR?'
+    )
+
+    panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel('label'),
+                FieldPanel('search_handler_location'),
+                FieldPanel('includes_ocr'),
+            ]
+        ),
+    ]
+
+    class Meta:
+        abstract = True
+
+
+class CollectionPageFacet(Orderable, CFacet):
+    """
+    Intermediate class for facets within a Collection Page.  
+    (needed to create an InlinePanel)
+
+    """
+    page = ParentalKey(
+        'lib_collections.CollectionPage', related_name="col_facet"
+    )
+
+
+class CBrowse(models.Model):
+    """
+    Class for cluster browses within a Collection Page.
+    """
+    label = models.CharField(max_length=255, blank=True)
+    include = models.BooleanField(
+        default=False, help_text='Include in sidebar?'
+    )
+    iiif_location = models.URLField(max_length=255, blank=True)
+
+    panels = [
+        MultiFieldPanel(
+            [
+                FieldPanel('label'),
+                FieldPanel('include'),
+                FieldPanel('iiif_location'),
+            ]
+        ),
+    ]
+
+    class Meta:
+        abstract = True
+
+
+class CollectionPageClusterBrowse(Orderable, CBrowse):
+    """
+    Intermediate class for cluster browses within a Collection Page.
+    (needed to create an InlinePanel)
+    """
+    page = ParentalKey(
+        'lib_collections.CollectionPage', related_name="col_cbrowse"
+    )
+
+
+class LBrowse(models.Model):
+    """
+    Class for list browses within a Collection Page.
+    """
+    label = models.CharField(max_length=255, blank=True)
+    include = models.BooleanField(
+        default=False, help_text='Include in sidebar?'
```

Comment:
Change help text to: Include browse term in collection sidebar? (Featured browse)